autolink/postlink/gradleLinker.js

```diff
@@ -69,6 +88,21 @@ class GradleLinker {
     return `"${DEFAULT_KOTLIN_VERSION}"`;
   }
 
+  /**
+   * Get the minSdkVersion value.
+   * @param { string } contents
+   */
+  _getMinSdkVersion(contents) {
+    var minSdkVersion = contents.match(/minSdkVersion = (\d*)/)
```

Comment:
We should account for zero or more spaces around to `=` symbol.
```suggestion
    var minSdkVersion = contents.match(/minSdkVersion\s=\s(\d*)/)
```

Reply:
Had to use `\s{0,}=\s{0,}` instead.